documentation/docs/14-configuration.md

```diff
@@ -53,6 +53,8 @@ const config = {
 		},
 		router: true,
 		serviceWorker: {
+			register: boolean,
+			scope: string,
```

Comment:
It might be better to make this be the whole options object so that users can set `type` and `updateViaCache` with this option as well https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register